scripts/motor-expert-screen

```diff
@@ -96,17 +96,20 @@ elif [ "${rtyp}" == 'motor' ]; then
     caget "$PREFIX:PN" > /dev/null 2>&1
     if [ $? -eq 0 ]; then
         /reg/g/pcds/package/epics/3.14//modules/pcds_motion/${OLD_VERSION}/launch-motor.sh "$PREFIX" > /dev/null 2>&1 &
-    elif [[ $PREFIX == *"MCS2"* ]]; then # smaracts have "motor" rtyp, so we catch them with the PREFIX
-        enc=$(caget $PREFIX.UEIP)
-        if [[ $enc == *"Yes"* ]]; then
-            edm -x -eolc -m "MOTOR=$PREFIX" mcs2_main.edl > /dev/null 2>&1 &
-        else
-            edm -x -eolc -m "MOTOR=$PREFIX" mcs2_openloop.edl > /dev/null 2>&1 &
-        fi
     else
-        cd /reg/neh/home/klg/epics/ioc/common/aerotech/current/motorScreens
-        #cd /reg/neh/home4/mcbrowne/trunk2/ioc/common/aerotech/current/motorScreens
-        edm -x -eolc -m "MOTOR=${PREFIX}" ens_main.edl >& /dev/null &
+	# smaracts have "motor" rtyp, so we catch them with the PREFIX or with smaract specific pvs
+	if [[ $PREFIX == *"MCS2"* || $(caget "$PREFIX:PTYPE_RBV" > /dev/null 2>&1) -eq 0 ]]; then
```

Comment:
PTYPE_RBV has a ton of matching PVs that demonstrate the issue, so I think this is a useful pick

Somewhat unrelated: does bash skip evaluation of the second part of the or statement if it can?

Reply:
Bash short-circuits && and || or statements - https://www.gnu.org/software/bash/manual/bash.html#Lists